Kinvey/Kinvey/TaskProgressRequest.swift

```diff
@@ -125,7 +125,7 @@ class TaskProgressRequest: NSObject {
         {
             let progressStatus = ProgressStatus(task)
             if (progressStatus.countOfBytesSent == progressStatus.countOfBytesExpectedToSend && progressStatus.countOfBytesReceived >= 0 && progressStatus.countOfBytesExpectedToReceive != 0) ||
-                (progressStatus.countOfBytesSent >= 0 && progressStatus.countOfBytesExpectedToSend > 0)
+                (progressStatus.countOfBytesSent >= 0 && (progressStatus.countOfBytesExpectedToSend > 0 || progressStatus.countOfBytesExpectedToSend == -1))
```

Comment:
`countOfBytesExpectedToSend` is `0` when the data is actually empty and it could can also be `-1` if the upload data is actually and unknown size stream